test/interpreter/function-divide.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function DIVIDE', () => {
+  it('should return #NA! error with the wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=DIVIDE(1)', '=DIVIDE(1, 1, 1)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('B1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate the correct value with correct defaults', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=DIVIDE(6,3)'],
+      ['=DIVIDE(,1)'],
+      ['=DIVIDE(,)']
```

Comment:
what is the default if the second argument is missing?

Reply:
it's not even default, it's standard null->number coercion, so 0

Reply:
Sure! I just used the "defaults" from the test name :) Can we have a tiny test for that: `DIVIDE(4,) -> DIV0` ?